samcli/local/lambdafn/config.py

```diff
@@ -51,13 +49,9 @@ def __init__(self, name, runtime, handler, code_abs_path, layers, memory=None, t
 
         if not isinstance(self.timeout, int):
             try:
-                self.timeout = ast.literal_eval(self.timeout)
-
-                # Guard against float values like "3.2"
-                if not isinstance(self.timeout, int):
-                    raise InvalidSamTemplateException("Invalid Number for Timeout: {}".format(self.timeout))
+                self.timeout = int(self.timeout)
```

Comment:
Thank you!

Reply:
How does this handle None / empty string?

Reply:
`int(None)` raises TypeError, `int('')` raises ValueError